LondonBritishLibrary/orient/BLorient619.xml

```diff
@@ -30,55 +30,43 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
                         <altIdentifier><idno>Wright cat. CCXXIX</idno></altIdentifier>
                     </msIdentifier>
 
-<msContents>
-
-<msItem xml:id="ms_i1">
-<locus from="1r"/>
-<title type="complete" xml:lang="gez" ref="LIT1297Dersan"/>
-<textLang mainLang="gez"></textLang>
-
-<msItem xml:id="ms_i1.1">
-<locus from="1r"/>
-<title type="complete" xml:lang="en">Discourses for the festival of the angel Raphael</title>
-<textLang mainLang="gez"></textLang>
-
-<msItem xml:id="ms_i1.1.1">
-<locus from="1r"/>
-<title type="complete" xml:lang="en" ref="LIT1297Dersan#HomilyJC">Homily of John Chrysostom</title>
-<textLang mainLang="gez"></textLang>
-</msItem>
-
-<msItem xml:id="ms_i1.1.2">
-<locus from="45r"/>
-<title type="complete" xml:lang="en" ref="LIT1297Dersan#Miracles">Miracles</title>
-<textLang mainLang="gez"></textLang>
-</msItem>
-</msItem>
-
-<msItem xml:id="ms_i1.2">
-<locus from="95r"/>
-<title type="complete" xml:lang="en">Discourses for the festival of the Four Beasts (8th of Ḫǝdār)</title>
-<textLang mainLang="gez"></textLang>
-
-<msItem xml:id="ms_i1.2.1">
-<locus from="95r"/>
-<title type="complete" xml:lang="gez" ref="LIT1297Dersan#HomilyRetua">Homily of Rǝtuʿa Haymānot</title>
-<incipit xml:lang="gez">ድርሳን፡ ዘ፬እንስሳ፡ ክቡራን፡ ዘደረሰ፡ ርቱዓ፡ ሃይማኖት፡ በእንተ፡ ኵሎሙ፡ ነገደ፡ መላእክት፡ ሕያዋነ፡ ሕይወት፡ እለ፡ ተፈጥሩ፡ እምእሳት፡ ወነፋስ፡
-ወለሰብአ፡ ጠባይዓ፡ ፬መላእክትሰ፡ ፪ወእሙንቱሰ፡ መዓርግ፡ ፱ዘይትነበብ፡ በሰላመ፡ እግዚአብሔር፡ አብ፡ አሜን።</incipit>
-<textLang mainLang="gez"></textLang>
-</msItem>
-
-<msItem xml:id="ms_i1.2.2">
-<locus from="106r"/>
-<title type="complete" xml:lang="gez" ref="LIT1297Dersan#HomilyChrysostom">Homily of John Chrysostom</title>
-<incipit xml:lang="gez">በእንተ፡ ፬እንስሳ፡ ዓበይት፡ መንፈሳውያን፡ አመ፳ለወርኃ፡ ኅዳር፡</incipit>
-<textLang mainLang="gez"></textLang>
-</msItem>
-
-</msItem>
-
-</msItem>
-</msContents>
+                    <msContents>
+                        
```

Comment:
`<summary/>` should preferably always be present, so that a summary is printed by the app (when no `<summary/>` is present, it cannot appear)